README.md

```diff
@@ -0,0 +1,52 @@
+# Shell Script
+
+This script (`bogons_sync-ripe.sh`) downloads the nro-delegated-stats file from RIPE and retrieves the IPv4/IPv6 prefixes and ASNs that are either available or reserved.
+Furthermore, it applies them to bird's default configuration file (bird.conf) as filters.
+
+## Prerequisites
+
+1. bc - An arbitrary precision calculator language. For Ubuntu: `apt-get install bc` 
+
+2. iprange - manage IP ranges. For Ubuntu: `apt-get install iprange` 
+
+## Use
+
+The script without any arguments (e.g. `./bogons-sync-ripe.sh`), outputs 4 files: ASN numbers, and IPv4/IPv6 prefixes in various formats, allowing the user to feed the script's output in the routing daemon of choice.
+
+If the script is run with parameter the config file of bird (e.g. `./bogons-sync-ripe.sh bird.conf`),
+it applies the outpout in bird's default configuration file in the form of filters.
```

Comment:
please add that it checks for the existence of the given filename (e.g bird.conf) at the default etc/ path of bird (thus: /etc/bird)

Reply:
Added